yt/frontends/boxlib/data_structures.py

```diff
@@ -888,10 +888,9 @@ def _setup1d(self):
         tmp = self.domain_dimensions.tolist()
         tmp.extend((1, 1))
         self.domain_dimensions = np.array(tmp)
-        tmp = list(self.periodicity)
-        tmp[1] = False
-        tmp[2] = False
-        self.periodicity = ensure_tuple(tmp)
+        self.periodicity = list(self.periodicity)
+        self.periodicity[1:] = False
```

Comment:
I think I don't understand this line easily, can you please clarify or add a comment? :)

Reply:
Thanks for reviewing this ! In light of recent developments (especially https://github.com/yt-project/yt/pull/2893, where I remove `ensure_tuple`), I just gave this another go. The specific line you're commenting here is now gone, and now periodicity should be only parsed once in `BoxlibDataset._parse_cparams`, in a dimensionality-agnostic way. Is it clearer now ?

Reply:
@ax3l We'll wait for your reply before this is merged :)

Reply:
> Is it clearer now ?

yes, thanks a lot.